spec/features/create_goobi_image_spec.rb

```diff
@@ -60,14 +60,18 @@
     click_button 'Log in'
 
     # find the new object
-    expect(page).to have_selector('h2', text: 'Home page')
+    expect(page).to have_text('Home page')
     click_link 'My tasks'
```

Comment:
noticed that it is now always in an `<h2>` tag (maybe depends on the version of goobi installed)...relax this expectation a bit